ctapipe/image/leakage.py

```diff
@@ -0,0 +1,59 @@
+"""
+Leakage calculation
+"""
+
+import numpy as np
+__all__ = ['leakage']
+
+
+def leakage(geom, image):
+    r"""Calculating the leakage-values for a given image.
+    Image must be cleaned for example with tailcuts_clean.
+
+    Parameters
+    ----------
+        geom: `ctapipe.instrument.CameraGeometry`
+            Camera geometry information
+        image: array
+            pixel values
+
+    Returns
+    -------
+        leakage_border_pixel1:   float
+            Number of shower-pixel on the border divided by all shower-pixel
+        leakage_border_pixel2:   float
+            Number of shower-pixel in the second row of the border
+             divided by all shower-pixel
+        leakage_border_photon1:   float
```

Comment:
I guess you mean photo-electrons and not photons here? It might be better to call it "intensity" as we do elsewhere.